lib/parser.js

```diff
@@ -311,8 +317,8 @@ module.exports = class Parser {
     // parens get treated as one word, if the contents aren't not a string.
     if (this.current.type === 'func' && this.current.unbalanced &&
         this.current.value === 'url' && this.currToken[0] !== 'string' &&
-        this.currToken[0] !== ')' && !this.options.loose) {
-
+        this.currToken[0] !== ')' && (checkForUrl(this.currToken[1]) || !this.options.loose)) {
```

Comment:
should the logic here be to automatically treat `url()` contents as a string? technically it should follow https://drafts.csswg.org/css-syntax-3/#consume-url-token for an unquoted string, if i'm reading it right (debatable) the alg is just to consume the content unless you hit a bad char or close parens? e.g. a `var()` in a url shouldn't be treated as an actual var